Makefile

```diff
@@ -1,16 +1,42 @@
-GFA_FILES := k.gfa DRB1-3123.gfa
+TEST_FILES := t k note5 overlap q.chop DRB1-3123 chr6.C4 LPA
+BASIC_TESTS := ex1 ex2
 GFA_URL := https://raw.githubusercontent.com/pangenome/odgi/ebc493f2622f49f1e67c63c1935d68967cd16d85/test
+GFA_ZIP_URL := https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/scratch/2021_05_06_pggb/gfas/chr8.pan.gfa.gz
 
-.PHONY: fetch test clean
-fetch: $(GFA_FILES)
+.PHONY: fetch og test clean test-all
+fetch: $(TEST_FILES)
 
-test: $(GFA_FILES)
-	turnt test/$^
+og: fetch $(foreach file,$(TEST_FILES),test/$(file).og) $(foreach file,$(BASIC_TESTS),test/basic/$(file).og)
 
-clean: 
-	rm -rf test/*.gfa
+test: og
+	-turnt --save --env baseline $(foreach file,$(TEST_FILES),test/$(file).gfa)
+	turnt $(foreach file,$(TEST_FILES),test/$(file).gfa)
+	-turnt --save --env baseline test/basic/*.gfa
+	turnt test/basic/*.gfa
 
-$(GFA_FILES): %.gfa:
+	-turnt --save --env baseline test/subset-paths/*.txt
+	-turnt test/subset-paths/*.txt
+
+test-all: GFA_ZIP_URL test/chr8.pan.og test
+	-turnt --save --env baseline test/chr8.pan.gfa
+	turnt test/chr8.pan.gfa
+
+clean:
+	rm -rf $(foreach file,$(TEST_FILES),test/$(file).*)
+
+	rm -rf test/basic/*.og
+	rm -rf test/basic/*.out
+
+	rm -rf test/subset-paths/*.out
+
+GFA_ZIP:
```

Comment:
I have changed it to make the `test/chr8.pan.gfa` file. I believe this should trigger because the `%.og` rule depends on `%.gfa`, and now it will only trigger if the `.gfa` file does not already exist.